### DISCUSSION TITLE
main windows mouse cursor flickering in game

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#21337] [Q&A]
### Type

Game behaviour

### Bug description

i randomly found the bug and it's pretty annoying that the windows cursor sometime flickering and being offset with the game cursor happened when you playing at fullscreen, lower resolution than native resolution (sr for the bad video quality)

### Screenshots or videos

https://user-images.githubusercontent.com/105227513/202914217-e15bb717-36f1-48c1-854b-385079793072.mp4

### Version

2022.1117.0

### Logs

[updater.log](https://github.com/ppy/osu/files/10050204/updater.log)
[database.log](https://github.com/ppy/osu/files/10050205/database.log)
[network.log](https://github.com/ppy/osu/files/10050206/network.log)
[performance.log](https://github.com/ppy/osu/files/10050207/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10050208/runtime.log)

**Comment by bdach** [comment]
* What input devices do you have plugged in? (A dxdiag report would also be nice.)
* "High precision mouse" on or off?
* In-game tablet handler on or off?
* Any custom drivers in use?

**Reply by ghost** [reply]
I use a wireless mouse
High precision mouse is on
no ingame tablet handler or any custom driver
(Dxdiag report)
[DxDiag.txt](https://github.com/ppy/osu/files/10055946/DxDiag.txt)
